src/WebJobs.Script.WebHost/Diagnostics/DeferredLoggerProvider.cs

```diff
@@ -48,54 +48,63 @@ public void ProcessBufferedLogs(IReadOnlyCollection<ILoggerProvider> forwardingP
                     await Task.Delay(_deferredLogDelay);
                 }
 
-                // Disable the channel and let the consumer know that there won't be any more logs.
```

Comment:
Was this a bug, Why was this moved inside?

Reply:
This isn't a bug, but we overlooked this scenario. Previously, we were closing the channel after initialization. With this update, we will continue listening to logs throughout the app's lifetime. The channel should only be closed if there are no logger providers to forward logs to.